src/identity/Flooq.Identity/Services/UserService.cs

```diff
@@ -0,0 +1,59 @@
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using Flooq.Identity.Data;
+using Flooq.Identity.Models;
+
+namespace Flooq.Identity.Services;
+
+/// <summary>
+/// Responsible for handling changes to the user accounts
+/// </summary>
+public class UserService : IUserService
+{
+  private readonly UserManager<ApplicationUser> _userManager;
+  private readonly FlooqIdentityContext _context;
+
+/// <summary>
+/// Responsible for handling changes to the user accounts
+/// </summary>
+  public UserService(FlooqIdentityContext context, UserManager<ApplicationUser> userManager)
+  {
+    _context = context;
+    _userManager = userManager;
+  }
+  /// <summary>
+  /// Get a user by Id
+  /// </summary>
+  /// <param name="userId">Of the user to return</param>
+  /// <returns>User with the given Id</returns>
+
+  public async Task<ApplicationUser> GetUsersById(string userId)
+  {
+    return await _context.Users.Where(u => u.Id.Equals(userId)).FirstOrDefaultAsync();
+  }
```

Comment:
Why FirstOrDefault? What would be the default value?

Reply:
The default value is null